rotkehlchen/types.py

```diff
@@ -849,9 +849,15 @@ class LocationAssetMappingDeleteEntry:
     @classmethod
     def deserialize(cls: type['LocationAssetMappingDeleteEntry'], data: dict[str, Any]) -> 'LocationAssetMappingDeleteEntry':  # noqa: E501
         """May raise:
-        -KeyError if required keys are missing
+        -DeserializationError if required keys are missing
         """
-        return cls(**data)
+        try:
+            return cls(
+                location=data['location'],
+                location_symbol=data['location_symbol'],
+            )
+        except KeyError as e:
+            raise DeserializationError(f'Missing key {e!s}') from e
```

Comment:
Where is. this used? Don't you need to adjust the error handled in other pkace?

Reply:
handled them in `rotkehlchen/api/v1/schemas.py` and `rotkehlchen/db/updates.py`
it's only used in these two placed